src/components/date_picker/date_picker_range.tsx

```diff
@@ -104,6 +120,8 @@ export const EuiDatePickerRange: FunctionComponent<EuiDatePickerRangeProps> = ({
           'euiDatePickerRange__start',
           startDateControl.props.className
         ),
+        onBlur: over([startDateControl.props.onBlur, onBlur]),
+        onFocus: over([startDateControl.props.onFocus, onFocus]),
```

Comment:
[super optional comment - feel free to collapse if you'd rather not address] I believe at some point we _may_ try to move away from lodash?... https://github.com/elastic/eui/issues/360

TBH I don't feel super strongly either way, but if you wanted to write this in vanilla-JS, we could do something natively like
```suggestion
        onBlur: () => {
          startDateControl.props?.onBlur?.();
          onBlur?.();
        },
        onFocus: () => {
          startDateControl.props?.onFocus?.();
          onFocus?.();
        },
```

Again totally up to you though, no worries if you'd rather keep it as-is!

Reply:
No problem; I wasn't aware of the move away from lodash. Originally I'd written the functions almost identically to how you've suggested, but changed to the lodash version for brevity. I've swapped it back!